packages/lodestar/src/chain/index.ts

```diff
@@ -5,3 +5,4 @@ export {IBeaconClock} from "./clock/index.js";
 export * from "./forkChoice/index.js";
 export * from "./initState.js";
 export * from "./stateCache/index.js";
+export * from "./regen/index.js";
```

Comment:
What do you need from regen? @wemeetagain we should make very clear to consumer that the API of Lodestar package is not guaranteed to be stable between releases. Otherwise we should carefully export only _some_ things.

Reply:
Do you really need to run `logParticipation`? That an internal tool we use to debug tests. It will be remove soon also. If you depend on such deep internals it will be costly to mantain on your side.

Reply:
Imo we should have an export strategy like:
- selectively export key modules at the top-level, they will have more stability and support guarantees between releases, consumers encouraged to use this
- star export all modules at the lower level, they will have zero stability or support guarantees between releases, consumers can use this at their own risk